setup.cfg

```diff
@@ -38,8 +38,9 @@ classifiers =
 packages = find:
 python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*
 install_requires =
-    setuptools  >= 30.0.0
-    pluggy >= 0.3.0, <1
+    importlib-metadata
```

Comment:
shouldn't we version guard from above?

Reply:
yessss I wasn't sure what to put here, I'll go with `>=0.12,<1`